lib/MetricsConsumer.js

```diff
@@ -61,12 +61,24 @@ class MetricsConsumer {
     _sendSiteLevelRequests(data) {
         const { type, site, ops, bytes } = data;
         if (type === 'completed') {
+            this._statsClient
+                .decrementKey(`${site}:${redisKeys.opsPending}`, ops);
+            this._statsClient
+                .decrementKey(`${site}:${redisKeys.bytesPending}`, bytes);
             this._sendRequest(`${site}:${redisKeys.opsDone}`, ops);
             this._sendRequest(`${site}:${redisKeys.bytesDone}`, bytes);
         } else if (type === 'failed') {
+            this._statsClient
```

Comment:
Greedy request, we will still have problems with backlog and I was hoping we could use the no-expire redis data to determine backlog.

Is it possible to have these global keys for backlog and separate keys for fails?

Reply:
I see your concern that backlog still does not factor in failures, and that it may be inaccurate (e.g. the case that the CRR operation completes in a later interval). I guess it's a matter of design philosophy—i.e. what is the reasoning behind using interval keys. Do we eventually want some kind of timeline history of events?